src/alpha-lab/README.md

```diff
@@ -130,6 +130,12 @@ export default {
           to: "/alpha-lab/gaze-contingency-assistive",
           img: "gaze-cont-main-page.png",
         },
+        {
+          title: "Blink and you'll miss it!",
```

Comment:
@tpfffr, I added this such that the article appears on the Alpha Lab landing page. Again, feel free to edit the title and/or text description